### DISCUSSION TITLE
Routing CV between 2...tracks in Reaper

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#420] [Q&A]
Hello,
Is there any possibility to make a CV/gate routing between two (or more) tracks in Reaper ? Each track with one Cardinal Plugins.
Nb: Without using midi infos ;-)

Many thanks.
Toph

**Comment by dromer** [comment]
I'm assuming you are using LV2? Does Reaper LV2 hosting support CV ports, and routing between them?

If not, this would be a feature request for Reaper. Would certainly be cool to have a routing matrix for cv i/o !

**Comment by subderground** [comment]
Hi, I always install the VST3 version of Cardinal.
I've just tried with the LV2 version and the CV routing between 2 Cardinals on 2 tracks works nicely in Reaper :-) 
With exactly the same settings, the VST3 version doesn't.

Btw, the LV2 version has glitches while resizing the window. It seems to have 2 handles layered in the bottom right corner; one works and the second glitches.

Thanks.

Ps: Yep, Reaper has a pin connector per track for routing.
![pin-connectors-reaper](https://user-images.githubusercontent.com/105162700/201514440-bcd3c7c2-221d-4180-8326-62fed4cf2526.png)

**Reply by dromer** [reply]
VST3 CV ports are very poorly supported, I'm surprised they even show up, but interesting to know of a DAW where this could be tested. This is in latest stable Reaper? (will have to try this when I get home)

**Comment by subderground** [comment]
Not the last one. 😊 But this feature has not been modified while  checking the changelog.
